runtime/onert/core/include/exec/train/ITrainableFunction.h

```diff
@@ -26,12 +27,21 @@ namespace exec
 namespace train
 {
 
+// Q: function 'extraExtraTensors' is not PURE virutal function, If so, Do we need to change this
+// class name? ITrainableFunction -> TrainableFunction
 class ITrainableFunction
 {
 public:
   virtual ~ITrainableFunction() = default;
   virtual void forward(bool training) = 0;
   virtual void backward() = 0;
+
+  // Implement this if extra tensor is needed
+  virtual backend::train::ExtraTensorRequests requestExtraTensors()
+  {
+    backend::train::ExtraTensorRequests r;
```

Comment:
(Q) Why should this object be created in every TrainableFunction?

Reply:
No, It isn't. 
Only PoolLayer, Conv, DConv, FC, BinaryArithmeticLayer requires ExtraTensors.

Reply:
I put more comments here : 

https://github.com/Samsung/ONE/pull/13486/files#r1706502494